.travis.yml

```diff
@@ -26,13 +29,6 @@ script:
 - tox -r
 
 deploy:
-  - skip_cleanup: true
-    provider: script
-    script: python scripts/release.py $TRAVIS_TAG
-    on:
-      branch: master
-      tags: true
-      condition: $TRAVIS_PYTHON_VERSION = "2.7"
   - skip_cleanup: true
     provider: script
     script: python scripts/release.py $TRAVIS_TAG
```

Comment:
Do you not still need the `tags: true` line here?  That was supposed to only deploy when we tagged a release rather than every commit to master

Reply:
I double checked and "tags: true" is still present for the 3.6 release, the 2.7 one was just completely gutted.

Reply:
Understood, got it, I missed that part.